osu.Game.Tests/Visual/Online/TestSceneMessageNotifier.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Online.Chat;
+using osu.Game.Overlays;
+using osu.Game.Users;
+
+namespace osu.Game.Tests.Visual.Online
+{
+    public class TestSceneMessageNotifier : OsuTestScene
+    {
+        private User friend;
+        private Channel publicChannel;
+        private Channel privateMesssageChannel;
```

Comment:
```suggestion
        private Channel privateMessageChannel;
```

Reply:
I should've figured the CI would fail, please rename references too thx.